webpack/__mocks__/foremanReact/components/common/Fill/GlobalFill.js

```diff
@@ -1,3 +0,0 @@
-
-const addGlobalFill = () => jest.fn();
-export default addGlobalFill;
```

Comment:
@laviro was this file deleted before? Looks like it is still used here https://github.com/Katello/katello/blob/0c44a2fa68e8fca1a1e6b86582512b603809d6d6/webpack/fills_index.js#L2

Reply:
oh right, that is from `foremanReact`, nevermind! :smiley:

Reply:
yeah, mocks are not needed anymore :)